src/data/zmod/crt_properties.lean

```diff
@@ -0,0 +1,93 @@
+/-
+Copyright (c) 2023 Ashvni Narayanan. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Ashvni Narayanan
+-/
+import data.zmod.basic
+import number_theory.padics.ring_homs
+
+/-!
+# Properties of ℤ/nℤ for the Chinese Remainder Theorem
+
+This file has some theorems regarding coercions with respect to the Chinese Remainder Theorem for
+`zmod n` for all `n`.
+
+## Main definitions and theorems
+ * `proj_fst`, `proj_snd`, `inv_fst`, `inv_snd` : lemmas regarding CRT
+
+## Tags
+zmod, CRT
+-/
+
+lemma proj_fst' {m n : ℕ} (h : m.coprime n) (a : zmod m) (b : zmod n) :
```

Comment:
I don't have time for a full review right now, but this definitely needs a longer name, as currently it mentions neither zmod nor Chinese remainder